crates/move-to-polka/src/linker.rs

```diff
@@ -119,6 +152,56 @@ pub fn create_blob(
     Ok(blob)
 }
 
+fn fetch_git_dep(
+    mapping: &mut Vec<String>,
+    dep_sources: &mut Vec<String>,
+    dep: &move_package::source_package::parsed_manifest::Dependency,
+    git_url: &str,
+) {
+    let path = Path::new("/tmp/move-deps");
```

Comment:
Is it cleaned somewhere?

Reply:
I'm not cleaning it ATM, assuming the OS will clean /tmp. That way subsequent test runs don't need to re-clone the repo.